aws/ec2/test_ec2_instance_running_required_amis.py

```diff
@@ -1,25 +1,39 @@
 import pytest
 
 from aws.ec2.helpers import ec2_instance_test_id
-from aws.ec2.resources import ec2_instances
+from aws.ec2.resources import ec2_instances, ec2_images_owned_by
+
+from datetime import datetime, timedelta
 
 
 @pytest.fixture
-def required_amis(pytestconfig):
-    return frozenset(pytestconfig.custom_config.aws.required_amis)
+def owned_amis():
+    return ec2_images_owned_by()
 
 
 @pytest.mark.ec2
 @pytest.mark.parametrize("ec2_instance", ec2_instances(), ids=ec2_instance_test_id)
-def test_ec2_instance_running_required_amis(ec2_instance, required_amis):
+def test_ec2_instance_on_acceptable_ami(ec2_instance, owned_amis,):
     """
-    Checks that all EC2 instances are running the required AMIs.
+    Checks that all EC2 instances are running on acceptable AMIs, meaning
+    an AMI that is no only than 180 and is owned by us.
     """
-    if len(required_amis) == 0:
-        pytest.skip("No required AMIs were provided")
-
-    assert (
-        ec2_instance["ImageId"] in required_amis
-    ), "EC2 Instance {0[InstanceId]} is running unexpected AMI: {0[ImageId]}".format(
-        ec2_instance
-    )
+    for tag in ec2_instance["Tags"]:
+        if tag["Key"] == "Name":
+            instanceName = tag["Value"]
+
+    nightyDaysAgo = datetime.now() - timedelta(days=180)
```

Comment:
We probably will want to make this configurable with 180 days as the default. I can make that change when I add the account ids for the amis request.